package.json

```diff
@@ -31,6 +31,7 @@
         "check:versions": "node utils/check_package_version_consistency.js",
         "build:docker": "docker build -t wot-servient .",
         "clean:dist": "npm exec  --workspaces -- npx rimraf tsconfig.tsbuildinfo dist",
+        "update:wot-typescript-definitions": "npx npm-check-updates  -u -f \"wot-typescript-definitions\" --deep",
```

Comment:
I wonder whether we want to have that. @erossignon you added a similar check for opcua in another PR.
@relu91 do you think this is fine?

Reply:
I find this very useful to consistently upgrade a particular module used everywhere in a mono repo.

Reply:
I see your arguments